apps/blockchain/test/blockchain/interface/account_interface/cache_test.exs

```diff
@@ -0,0 +1,127 @@
+defmodule Blockchain.Interface.AccountInterface.CacheTest do
+  use ExUnit.Case, async: true
+
+  alias Blockchain.Interface.AccountInterface.Cache
+
+  alias Blockchain.Interface.AccountInterface.Cache
+
+  describe "update_current_value/4" do
+    test "sets current_value" do
+      cache = %Cache{cache: %{}}
```

Comment:
I don't think you need `cache: %{}` inside. That's the default for the `Cache` struct, right? You should be able to just do `%Cache{}`.